src/forwarder.spec.ts

```diff
@@ -35,6 +36,59 @@ function awsMock(existingSubs: { [key: string]: any }, stackName?: string): Aws
   } as Aws;
 }
 
+describe("canSubscribeLogGroup",() => {
+  it("tells addCloudWatchForwarderSubscriptions function it can subscribe to a given log group with 0 existing subscription filters", async() => {
```

Comment:
For all of these tests, I would remove the part where you say "tells addCloudWatchForwarderSubscriptions function...".

It's true that the `canSubscribeLogGroup()` function is used right now by the `addCloudWatchForwarderSubscriptions()` function, but this could be refactored or renamed in the future. So it would be better to really just focus in on what this function does in isolation. That's why we call them "unit tests" -- they test one unit (function) and that unit only.

So in this case I would phrase the tests as "Returns true if log group has 0 existing subscription filters", and the same pattern for the other tests. What we do after this function returns true or false is something that can be tested elsewhere -- these tests just focus on whether this one function returns true or false.